templates/invitation.html

```diff
@@ -1,5 +1,15 @@
-<h2>{{name}}</h2>
+<p>Hello {{name}}</p>,
 
-<p>You have an invitation to join {{establishment}}</p>
+<p>You have an invitation you to join {{establishment}}.</p>
 
-<p><a href="https://public-ui.notprod.asl.homeoffice.gov.uk/accept?token={{token}}">Accept this invitation</a></p>
\ No newline at end of file
+<p>To accept this invitation, please follow the link below:<br />
+  <a href="{{acceptLink}}">{{acceptLink}}</a>
+</p>
+
+<p>This link will stop working 7 days after you've received it.<br />
+This is to keep the service secure.</p>
+
+<p>This is an automated email. Please do not reply. If you need help, send an email to <a href="mailto:animalscience@digital.homeoffice.gov.uk">animalscience@digital.homeoffice.gov.uk</a>.</p>
```

Comment:
Emails are sent from that address anyway, let's not be so hostile to our users. Replies are fine.

Reply:
don't shoot the messenger

Reply:
_copy paste_, but yeah, agree